frontend/openapi-codegen.sh

```diff
@@ -0,0 +1,15 @@
+# Clean slate
+rm -rf ./src/codegen
+
+npx @openapitools/openapi-generator-cli generate \
+  -i http://localhost:8000/openapi.json -o src/codegen \
+  -g typescript-axios --additional-properties=supportsES6=true
+
+# Tidy up unnecessary artifacts
+# Not interested in publishing as a separate npm package
```

Comment:
Wondering if there might be settings for this.

In general, I'm a bit split, if we even want to check this in. We could also just gitignore it and run it as a pre-start script. But it's not as explicit when something changes that way.

Reply:
I thought there may be a setting also, but doesn't seem so in [the documentation](https://openapi-generator.tech/docs/generators/typescript-axios).

Reply:
Actually, I think there is a global property for this rather than generator specific. Will try that out.

Reply:
I've slightly improved this using the ignore file in the output directory. Less file churn that way too. We could .gitignore the remaining artifacts that we don't care for which are un-skippable.

Reply:
I'm not sure which ones you mean, I can't see anything in that folder that seems useless now

Reply:
The remaining files this script is still removing

Reply:
I think we should probably keep those around, they seem like cache/version info, so might be relevant on updates?